i18n/en.json

```diff
@@ -1815,6 +1815,53 @@
   "admin.reload.reloadDescription.featureName": "Reload Configuration from Disk",
   "admin.reload.reloadDescription.recycleDatabaseConnections": "Environment > Database > Recycle Database Connections",
   "admin.reload.reloadFail": "Reloading unsuccessful: {error}",
+  "admin.reporting.workspace_optimization.access.description": "Web server settings could be affecting access.",
+  "admin.reporting.workspace_optimization.access.site_url.description": "Your webserver settings are not passing a live URL test, this would prevent users from accessing this workspace, we recommend updating your settings.",
+  "admin.reporting.workspace_optimization.access.site_url.title": "Misconfigured Web Server",
+  "admin.reporting.workspace_optimization.access.title": "Workspace Access",
+  "admin.reporting.workspace_optimization.chip_problems": "Problems: {count}",
+  "admin.reporting.workspace_optimization.chip_suggestions": "Suggestions: {count}",
+  "admin.reporting.workspace_optimization.chip_warnings": "Warnings: {count}",
+  "admin.reporting.workspace_optimization.configuration.description": "You have configuration problems to resolve",
+  "admin.reporting.workspace_optimization.configuration.ssl.title": "Configure SSL to make your server more secure",
```

Comment:
Is the en.json file missing a string for the ssl description? "Make your server more secure by configuring SSL. [Learn more]"

The [Learn more] link would take the user to https://docs.mattermost.com/onboard/ssl-client-certificate.html

Reply:
@cwarnermm So this one in my opinion isnt just a "nice to have" its a scenerio where if they dont have SSL configured, it should be a bit more of a heavy recommendation..  I feel like it sounds kind of optional the way you have it phrased where we should be more heavily recommending... Ultimately its a really bad idea to not have SSL  configured for a production environment.. and is likely only not configured as a left over from testing...